### PR TITLE
forge-gui-desktop: Reorganize tests (forge.card)

### PR DESCRIPTION
**Pull request by allentiak**

### CODE REVIEW

**Review by Agetian — DISMISSED** [review]

File: forge-gui-desktop/src/test/java/forge/deck/DeckRecognizerTest.java [hunk 1]
```diff
@@ -44,69 +44,67 @@ private void initMaps(){
...
-     * - Collector Number
-     * - Deck Name
-     * - Card Type
-     * - Deck Section
-     * - Mana Colour
-     * ======================================
+    /*
+     * ====================================== TEST SINGLE (Card and Non-Card) Parts
+     * - CardName - Card Amount - Set Code - Collector Number - Deck Name - Card
+     * Type - Deck Section - Mana Colour ======================================
```
Comment (Hanmac) [inline forge-gui-desktop/src/test/java/forge/deck/DeckRecognizerTest.java:50]: i think these doc lines shouldn't be compressed?

Comment (Agetian) [reply forge-gui-desktop/src/test/java/forge/deck/DeckRecognizerTest.java:50]: Agreed

Comment (allentiak) [reply forge-gui-desktop/src/test/java/forge/deck/DeckRecognizerTest.java:50]: Could you please point me to a correct/acceptable example?
This PR is mostly a succession of automated refactorings...

Comment (allentiak) [reply forge-gui-desktop/src/test/java/forge/deck/DeckRecognizerTest.java:50]: @Hanmac BTW: Thanks for your detailed review!

Comment (Hanmac) [reply forge-gui-desktop/src/test/java/forge/deck/DeckRecognizerTest.java:50]: before the Doc Lines showed some Kind of list, now after the auto refactor it doesn't anymore

Comment (allentiak) [reply forge-gui-desktop/src/test/java/forge/deck/DeckRecognizerTest.java]: Thanks. Fixed.

**Review by Agetian — APPROVED** [review]

**Pull request merged**